CHANGES.rst

```diff
@@ -33,9 +34,11 @@ Fixes:
 
 Changes:
 --------
-- Add reference link to ReadTheDocs URL of `Weaver` in API landing page.
-- Add references to `OGC-API Processes` requirements and recommendations for eventual conformance listing
+- Add reference link to ReadTheDocs URL of ``Weaver`` in API landing page.
+- Add references to ``OGC-API Processes`` requirements and recommendations for eventual conformance listing
```

Comment:
these two items should remain single quote for italic terms because they are not code